### PR TITLE
tools/cephfs-shell: add support for batch file processing and execute commands from arguments.

### PR DESCRIPTION
**Pull request by rpavani1998**
Features to cephfs-shell:
1) to process a batch file instead of interactive commands.
2) execute commands from arguments

Fixes:http://tracker.ceph.com/issues/26853
Fixes:http://tracker.ceph.com/issues/26855
Signed-off-by: Pavani Rajula <rpavani1998@gmail.com>

- [x] References tracker ticket
- [x] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: src/tools/cephfs/cephfs-shell [hunk 1]
```diff
@@ -812,9 +812,16 @@ class CephFSShell(Cmd):
  
 if __name__ == '__main__':
     config_file = ''
-    if sys.argv[1] == '-c':
-        config_file = sys.argv[2]
+    main_parser = argparse.ArgumentParser(description = '')
```
Comment (tchaikov) [inline src/tools/cephfs/cephfs-shell]: could you run `pep8` or `pycodestyle` against the new code, just to make sure we don't have more style warnings/errors ?

**Comment by tchaikov** [comment]
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes

File: src/tools/cephfs/cephfs-shell (hunk 1 above)
Comment (rpavani1998) [reply src/tools/cephfs/cephfs-shell]: @tchaikov Do I make this in a seperate pr?

Comment (tchaikov) [reply src/tools/cephfs/cephfs-shell]: @rpavani1998 i don't think it's necessary. would be better to fix the
```
E251 unexpected spaces around keyword / parameter equals
```
in this very commit.

i just wanted to raise your attention here. as i am trying to get a pep8-clean cephfs-shell before it's packaged. and i am posting #23480, so probably you can (early) review it. i will test it after it's pep8 clean.

**Comment by rpavani1998** [comment]
@batrick  I will look into it.  I would like to make all such fixes into another pr.

**Comment by batrick** [comment]
@rpavani1998 please rebase this PR, make sure it is pep8 clean, and incorporate the fixes I sent to you in the email last week.

**Pull request merged**